plugins/dummy_sha256_sdk/src/main.rs

```diff
@@ -14,6 +15,13 @@ fn sha256(content: &[u8]) -> Vec<u8> {
 	hasher.finalize().to_vec()
 }
 
+#[query(default = false)]
+fn altsha256(_engine: &mut PluginEngine, content: Vec<u8>) -> Result<Vec<u8>> {
```

Comment:
this function should need to be `async` so it *could* call `engine::query` if need be. 

also, replace use of `sha256` and its `Query` impl. rename alt func to `sha256`.